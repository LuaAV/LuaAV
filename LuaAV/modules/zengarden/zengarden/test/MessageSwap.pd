#N canvas 569 334 450 300 10;
#X obj 218 30 loadbang;
#X obj 289 30 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X obj 218 176 swap;
#X obj 218 68 t b b b b b;
#X msg 361 118 42 7;
#X msg 300 119 42 7 3 4;
#X msg 265 120 66;
#X msg 233 121 77;
#X obj 218 216 print left;
#X obj 294 215 print right;
#X obj 82 129 swap 12;
#X obj 82 207 print twelve;
#X obj 121 181 print nothing;
#X connect 0 0 3 0;
#X connect 0 0 10 0;
#X connect 1 0 3 0;
#X connect 1 0 10 0;
#X connect 2 0 8 0;
#X connect 2 1 9 0;
#X connect 3 0 2 0;
#X connect 3 1 7 0;
#X connect 3 2 6 0;
#X connect 3 3 5 0;
#X connect 3 4 4 0;
#X connect 4 0 2 0;
#X connect 5 0 2 0;
#X connect 6 0 2 1;
#X connect 7 0 2 0;
#X connect 10 0 11 0;
#X connect 10 1 12 0;
