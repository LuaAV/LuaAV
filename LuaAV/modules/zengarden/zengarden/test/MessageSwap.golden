[@ 0.000ms] right: 42
[@ 0.000ms] left: 7
[@ 0.000ms] right: 42
[@ 0.000ms] left: 7
[@ 0.000ms] right: 77
[@ 0.000ms] left: 66
[@ 0.000ms] right: 77
[@ 0.000ms] left: 66
[@ 0.000ms] nothing: 0
[@ 0.000ms] twelve: 12